lib/dependabot/file_parsers/java/maven.rb

```diff
@@ -9,7 +9,8 @@ module Dependabot
   module FileParsers
     module Java
       class Maven < Dependabot::FileParsers::Base
-        DEPENDENCY_SELECTOR = "dependencies > dependency, plugins > plugin"
+        DEPENDENCY_SELECTOR = "parent, dependencies > dependency,
+                               plugins > plugin"
```

Comment:
Here I've split over multiple lines in order to satisfy Rubocop, dunno if this works but the tests are OK.

Reply:
Should be fine - css and xpaths treat all whitespace as equivalent (and your new test is passing so it's definitely fine).